src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ CHANGELOG
  * Deprecated the `web_profiler.position` config option (in 4.0 version the toolbar
    will always be displayed at the bottom) and the `web_profiler.debug_toolbar.position`
    container parameter.
+ * added ability to auto-refresh the toolbar if a `Symfony-Debug-Toolbar-Replace` header is set
+   with any JavaScript truthy value
```

Comment:
As headers are always strings, the only falsy value is the empty string (try `console.log(!!'0')`).
So this is confusing especially given that you could use the *number* `0` when setting the header (which is a Javascript falsy value) without realizing it will be casted to the `'0'` string by PHP when setting the header (which is a Javascript truthy value)

Reply:
`!!parseInt()` will do.

Reply:
Should I change this to "any non-empty string"? Or should something change with the functionality? I considered using a predefined string, but forcing a string 'true' doesn't make sense and '1' doesn't really either, I concluded that leaving it undefined would be better.

Reply:
@ro0NL `!!parseInt()` would fail for a string of "true" (`!!parseInt('true') === false`)

Reply:
Well, I think using `'1'` and `'0'` (well, `'1'` or anything else probably) would be the simpler to document (set the `Symfony-Debug-Toolbar-Replace` header to `1` to replace the toolbar), and it would also be consistent with the way we handle env variables (which are also string only) meant to contain a boolean (we use `'0'` to represent `false`).